.github/workflows/ci.yml

```diff
@@ -6,13 +6,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.7', '3.0', '3.1', '3.2', 'head', 'truffleruby' ]
-        rails: [ '6.1', '7.0', 'main' ]
-        include:
```

Comment:
Just looking at this briefly, I don't understand what the purpose of this `include` is yet.

It was updated in #153 to remove all of the old versions, but curious as to why it was removed here if you have ideas.

Reply:
👋  I think, based on my assumption, but I could also be wrong that this was a left over from previous commits.

**More specific:**
The includes for Ruby 3.2 was added early here: https://github.com/rails/globalid/commit/8b2fdd6c183db7d5460fae691c31dc79aeb54b27 when there was still support for Rails 5.1 (which does not work with Ruby 3.2 and made sense 3.2 to be explicit).

Then later Rails 5.1 was removed but that include was still left there due to the reason that `ruby: [...]` did not had 3.2.

Then, when 3.2 was added in the main `ruby: [...]`  https://github.com/rails/globalid/commit/ecc0adf39e3dac50881642163c1d39ce654ee0d4 which by that time there was no Rails 5.1 it would make sense to also remove this specific includes as there were not really adding any new matrix combination.

For the current PR here, I thought it would make sense to remove those extra includes since they are not adding any new jobs as it is. 

Reference also on [Github CI docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations)

Reply:
Just eyeballing the two workflows from this branch and main, I can't tell the difference 🤔

Reply:
Hi @zzak thanks for the response, I am not sure I am following. 
Do you mean the actual workflows? 🤔 

On main we have a total of 18 checks:
![Screenshot 2023-12-29 at 08 39 59](https://github.com/rails/globalid/assets/1176797/78d4964e-bf5b-4075-903a-db5861644095)

Which is 6 Ruby flavours * 3 Rails flavours 

While on this branch:
 7 Ruby flavours * 4 Rails flavours  == 28 runs as It adds Rails 7.1 and Ruby 3.3
![Screenshot 2023-12-29 at 08 39 50](https://github.com/rails/globalid/assets/1176797/9f230cb2-afa0-4829-af79-e794b0fcfafd)

Reply:
Yeah, I mean not counting the new workflows you've added, it seems like there is no difference without the `include` which makes me think it's fine to be removed. 🤔

Reply:
Ah, yes! As I mentioned on my first response, based on how the main branch is now that include was not making any difference, just like you said, hence the cleanup.🙂

Reply:
I've given it a look and like I said, the workflows look fine so I don't think there is anything actionable, just wait for a review. :pray:

Maybe you could squash your commits?

Reply:
Awesome, commits look squashed ✨ !